src/DurableTask.Netherite/StorageLayer/Faster/AzureBlobs/BlobManager.cs

```diff
@@ -134,18 +135,34 @@ public class FasterTuningParameters
             {
                 LogDevice = this.HybridLogDevice,
                 ObjectLogDevice = this.ObjectLogDevice,
-                PageSizeBits = pageSizeBits,  
+                PageSizeBits = pageSizeBits,
                 MutableFraction = tuningParameters?.StoreLogMutableFraction ?? 0.9,
-                SegmentSizeBits = tuningParameters?.StoreLogSegmentSizeBits ??
-                    (useSeparatePageBlobStorage ? 35   // 32 GB
-                                                : 32), // 4 GB
+                SegmentSizeBits = segmentSizeBits,
                 PreallocateLog = false,
                 ReadFlags = ReadFlags.None,
                 ReadCacheSettings = null, // no read cache
                 MemorySizeBits = memorybits,
             };
         }
 
+        public static (int pageSizeBits, int segmentSizeBits) GetImmutableEventLogParameters(bool useSeparatePageBlobStorage, FasterTuningParameters tuningParameters)
+        {
+            int pageSizeBits = tuningParameters?.EventLogPageSizeBits ?? 21; // 2MB
```

Comment:
My bit-arithmetic fails me here. How are you going from 21 bits (?) to 2MB?

Reply:
2^21 bytes = (2^1)(2^10)(2^10) bytes = 2 * 1024 * 1024 bytes = 2 MB

Reply:
I should have picked up on this 😅  Thanks!